config/custom.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Custom Configuration
+ *
+ * https://craftcms.com/docs/4.x/config/#custom-settings
+ */
+
+return [
+    // Global settings
+    '*' => [
+        'enableGoogleAnalytics' => false,
```

Comment:
Done.

Reply:
ID check should be here, in the default config so that if we ever want to test GA on dev or staging we can do that by adding the key to the `.env`.

On production we _always_ want it to be enabled, so there it should always be `true`. The code snippet in `html.twig` will do the check of verifying if a GA code is present or not before rendering the snippet. Remember, in a project the GA4 code might come from the control panel and not the `.env` file.

Reply:
I thought it should be `false` by default and a check is needed for production as we need tracking to be enabled for production only. Hadn't thought of the testing GA for the `dev` environment case.

Updated now.